### PR TITLE
Add thanks for coming news post

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+:template: 2016/eu.html
+
+Thanks for coming to Write the Docs Europe 2016
+===============================================
+
+Hi everyone,
+
+Thanks for attending our third European Write the Docs conference in Prague! We're so grateful to our sponsors, speakers, volunteers, and attendees for making the conference possible.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: I'd feel better if we didn't write sponsors first ;) so something like "speakers, volunteers, sponsors, and attendees"

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+:template: 2016/eu.html
+
+Thanks for coming to Write the Docs Europe 2016
+===============================================
+
+Hi everyone,
+
+Thanks for attending our third European Write the Docs conference in Prague! We're so grateful to our sponsors, speakers, volunteers, and attendees for making the conference possible. 
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that is forming. Below we've included all of the resources you might need for staying in touch, like the Write the Docs Slack. We also have a short anonymous survey for you to fill out, so we can make the conference even better next year.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: "like the WTD slack" doesn't feel like it's necessary here, we can just keep the sentence short

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Tell us how we did
+
+Let us know how we can improve for future conferences and gatherings. We really appreciate your candid feedback!
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md:14]: is candid=anonymous? maybe we can explicitely use "anonymous" for the benefits of our non-natives

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+**Our three question survey is quick and easy, so [go fill it out](https://goo.gl/forms/RQE7qyaFvaMNP2Zo2).**
+
+## Speak, Sponsor, or Organize next year
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: capitalization Inconsistency? :)

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+This is the end of our 2016 round of conferences,
+but that doesn't mean things are over!
+We'll be having another North American and European conference next year.
+
+We're always looking for folks who want help us organize the events.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: the events > events (to imply more than just the existing events)

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+We have worked to document the [roles](http://www.writethedocs.org/organizer-guide/confs/event-roles/) that need to happen to put the conference together.
+We could use a few folks who are interested in lending a hand to make the event happen,
+so please let us know if that's you!
+
+We also depend on sponsors to make our events as magical as they are,
+while keeping ticket prices low.
+If you are curious about sponsoring Europe or North America next year,
+please contact us at [sponsorship@writethedocs.org](mailto:sponsorship@writethedocs.org).
+
+Also don't forget that inspiration and start thinking of your topic ideas for next year.
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: "that inspiration"?

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+Thanks to [Kay Smoljak](https://twitter.com/goatlady) for putting them together so quickly.
+They are licensed for non-commercial use with attribution,
+so feel free to use them in write-ups about the event.
+
+All the videos for the conference will be posted soon.
+We'll send another update once they're available!
+
+## Write the Docs Slack
+
+The Write the Docs Slack has really taken off this year! Our general Slack channel has been extra busy in the days and weeks leading up to the conference, and we hope that y'all continue the conversations throughout the year. If you haven't joined, head on over and join us! We're at [Write the Docs Slack](http://slack.writethedocs.com/).
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: is it slack.writethedocs.com or writethedocs.slack.com or slack.writethedocs.org or slack.slack.slack.slack?

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+## Meetups
+
+We hope you continue to [build the community](http://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](http://www.writethedocs.org/meetups/), consider [starting](http://www.writethedocs.org/meetups/starting/) a meetup!
```
Comment (thatdocslady) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: "across the country" sounds like leftovers from NA, maybe we can rephrase for global?

**Review by thatdocslady — COMMENTED** [review]
GitHub's new review workflow is awesome!

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 1 above)
Comment (plaindocs) [reply docs/conf/eu/2016/news/thanks-for-coming.md]: attendees, speakers, volunteers and sponsors? :-p

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 2 above)
Comment (plaindocs) [reply docs/conf/eu/2016/news/thanks-for-coming.md]: so we can keep improving the conference?

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 6 above)
Comment (plaindocs) [reply docs/conf/eu/2016/news/thanks-for-coming.md]: Not sure about "roles that need to happen". Maybe roles that are essential to putting on a great conference? Or roles that are necessary?

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+so please let us know if that's you!
+
+We also depend on sponsors to make our events as magical as they are,
+while keeping ticket prices low.
+If you are curious about sponsoring Europe or North America next year,
+please contact us at [sponsorship@writethedocs.org](mailto:sponsorship@writethedocs.org).
+
+Also don't forget that inspiration and start thinking of your topic ideas for next year.
+We'll be opening our North American CFP in December,
+and Europe will be open in April next year.
```
Comment (plaindocs) [inline docs/conf/eu/2016/news/thanks-for-coming.md:36]: Do we want to talk about Portland CFP not NA? (applies throughout?) Also:

> We'll be opening our North American CFP in December, and our European CFP in April next year.

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 7 above)
Comment (plaindocs) [reply docs/conf/eu/2016/news/thanks-for-coming.md]:
```
[Join](http://slack.writethedocs.org/) our slack channel, or [sign back in](https://writethedocs.slack.com/messages)!
```

maybe?

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 8 above)
Comment (plaindocs) [reply docs/conf/eu/2016/news/thanks-for-coming.md]: "across THE WORLD"! DOMINATE!

File: docs/conf/eu/2016/news/thanks-for-coming.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+We hope you continue to [build the community](http://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](http://www.writethedocs.org/meetups/), consider [starting](http://www.writethedocs.org/meetups/starting/) a meetup! 
+
+Keep us posted if you start one (or encounter a new one) so that we can spread the word.
+
+## Thanks again
+
+The conference is possible only because folks like you make it a great experience.
+Thanks for making it another wonderful year.
+We hope to see you back again next year,
+or even sooner at one of our other events that happen throughout the year.
```
Comment (plaindocs) [inline docs/conf/eu/2016/news/thanks-for-coming.md]: > We hope to see you back again next year, or even sooner at one of our regular local meetups.

**Review by plaindocs — COMMENTED** [review]
BOOM!

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 7 above)
Comment (ericholscher) [reply docs/conf/eu/2016/news/thanks-for-coming.md]: slack.wtd.[org|com] is the signup :)

File: docs/conf/eu/2016/news/thanks-for-coming.md (hunk 9 above)
Comment (ericholscher) [reply docs/conf/eu/2016/news/thanks-for-coming.md:36]: Yea, just seemed inconsistent to say "Portland and European" -- it works if we do "Portland and Prague", but I think keeping them both either city or geographical works.

**Pull request merged**